lib/sasl/src/systools_make.erl

```diff
@@ -1904,8 +1904,10 @@ del_tar(Tar, TarName) ->
     file:delete(TarName).
 
 add_to_tar(Tar, FromFile, ToFile) ->
-    case erl_tar:add(Tar, FromFile, ToFile, [compressed, dereference]) of
+    case catch erl_tar:add(Tar, FromFile, ToFile, [compressed, dereference]) of
```

Comment:
Do you have a good the reason for the `catch` here?

Seems that it is unnecessary. If it necessary, we prefer `try` in new case.

@sirihansen Any thoughts?